Testing/UnitTests/xASL_ut_function_xASL_adm_CorrectName.m

```diff
@@ -0,0 +1,80 @@
+function UnitTest = xASL_ut_function_xASL_adm_CorrectName(TestRepository)
+%xASL_ut_function_xASL_adm_CorrectName Individual unit test for xASL_adm_CorrectName
+%
+% INPUT:        TestRepository - Path to test repository.
+%
+% OUTPUT:       UnitTest  - Test structure
+%               name      - Name of tested module or submodule (char array)
+%               unit      - Insert one of the following: 'Module', 'Submodule' or 'Function'
+%               passed    - Result of all subtests combined (true or false)
+%               test      - Structure with individual subtest results
+%
+% -----------------------------------------------------------------------------------------------------------------------------------------------------
+% DESCRIPTION:  Should be run using xASL_ut_UnitTesting.
+%
+% EXAMPLE:      UnitTests(1) = xASL_ut_function_xASL_adm_CorrectName(TestRepository);
+% -----------------------------------------------------------------------------------------------------------------------------------------------------
+% Copyright 2015-2021 ExploreASL
+
+
+%% Test run 1
+
+% Give your individual subtest a name
+UnitTest.tests(1).testname = 'Simple text example';
+
+% Start the test
+testTime = tic;
+
+% Run your test here
+strOut = xASL_adm_CorrectName('abc$%&def');
```

Comment:
you want to have some backslashes and umlauts - these were a horror

Reply:
1b1f95f